### ISSUE TITLE
HTML Elements not nesting correctly

### ISSUE THREAD

**Issue by ghost** [Nanakii/summernote-plugins#1]
Hey thanks for the plugin this was exactly what I was looking for. However had one issue which was bothering me as I could not nest multiple HTML components where the cursor is currently placed.

Seems there is an issue with the pasteHTML method which keeps messing thing up i.e. could not nest multiple Bootstrap templates such as containers and stuff.

Anyway, though I would share my hack to get things nesting correctly encase anyone else needs this functionality.

I updated the following:

```
$.get(path)
    .done(function (data) {
        var node = document.createElement('span');
        node.innerHTML = data;
        context.invoke('editor.insertNode', node);
    }).fail(function () {
    alert('template not found in ' + path);
});      
```

**Comment by celestialaly** [comment]
Hey there !

I'm glad to have some feedback :)
Indeed I looked at your issue and noticed the problem.

Since your modification is perfectly working, would you mind making a PR ?
I will merge it right after.

Thanks !

**Comment by ghost** [comment]
Yeah no problem I can give a shot.

**Comment by celestialaly** [comment]
fixed in #2
